client/src/components/hero/Hero.tsx

```diff
@@ -18,6 +27,42 @@ interface HomeProps {
  * Hero for main home page.
  */
 const UnstyledHero: FunctionComponent<HomeProps> = ({ className }) => {
+  const [announcements, setAnnouncements] = useState<Announcement[]>();
+
+  // Get Announcement data.
+
+  const announcementDataQuery = useQuery(GET_HERO_ANNOUNCEMENTS);
```

Comment:
This is unnecessary - you should use `HomeDataViewer` query to retrieve these.

Reply:
Also we explictly talked about this during one of the meeting to add it into homeViewerQuery